### ISSUE TITLE
Corrupted apk with feature/#9CI

### ISSUE THREAD

**Issue by corus87** [kalliope-project/kalliope-app#23]
Hi, 
don't know if this bug is known but when I build the apk with "ionic cordova build android --release" the apk is corrupted and I can't install it. 
But without the --release flag I get a working APP.

Anyway great job on the APP I like the new features very much.
Thanks a lot for your work!

**Comment by LaMonF** [comment]
I will have a look on the release. We did not try to release this version yet :). But it will come very soon.
I just give a quick try and I manage to build the unsigned APK.

Is it failing while building the APK or when you try to push it to on your Phone ?
Which version of ANDROID SDK are you using on your phone ?

**Comment by corus87** [comment]
I can also build the apk without any errors, it just failing when I try to install it on my phone with "App not installed package appears to be corrupted". 
Android SDK on my phone? I set everything up on a fresh Ubuntu 18.04 just to build the app, I used the latest version of Android SDK and the recommended versions of the requirements given in the readme. 
After I google I found [this](https://forum.ionicframework.com/t/release-produces-corrupted-apk/94611/28) where someone tried it without the --release flag and it worked also in my case.
